### PR TITLE
Make Travis run tests

### PR DESCRIPTION
**Pull request by ayaleloehr**
Adds rubocop, bundle-audit, specs, and code coverage enforcement to Travis

### CODE REVIEW

File: va_rx.gemspec [hunk 1]
```diff
@@ -36,4 +36,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "webmock", "~> 2.1.0"
   spec.add_development_dependency "vcr"
   spec.add_development_dependency "pry"
+  spec.add_development_dependency "brakeman"
+  spec.add_development_dependency "bundler-audit"
+  spec.add_development_dependency "rubocop"
```
Comment (saneshark) [inline va_rx.gemspec]: are these being run automatically by Travis?

Comment (ayaleloehr) [inline va_rx.gemspec]: Not yet. but they will be soon once i get that working and commit... still a WIP here. Should have added that to the PR title (will do it now!)

**Comment by ayaleloehr** [comment]
@saneshark lots of commits here, but turned into a pretty small in the end actually. PTAL!

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -6,3 +6,8 @@
 require 'va_rx'
 
 RSpec.configure
+
+require 'simplecov'
```
Comment (saneshark) [inline spec/spec_helper.rb]: What if we wrap this up in an environment variable that only when passed runs coverage, ulternately an environmental variable that tells it not to run.

```ruby
unless NOCOVERAGE=true
   require 'simplecov'
   ...
end
```

**Comment by saneshark** [comment]
LGTM other than the nit pick comment above.

File: spec/spec_helper.rb (hunk 2 above)
Comment (ayaleloehr) [inline spec/spec_helper.rb]: I like the idea of an environment variable that tells it not to run. Then the default would be to have it run, but to speed up local tests (or for any other reason) anyone could set that ENV variable and skip simplecov.

Comment (saneshark) [inline spec/spec_helper.rb]: Yea, and that has the added benefit of not requiring changes to any other files, ie. travis.yml

**Comment by ayaleloehr** [comment]
Not a nit, a useful comment!  (and in fixing I created a lint error, which broke Travis, which shows this PR is working!)  Addressed and merging since it has a LGTM

**Pull request merged**